classification/classification_network.py

```diff
@@ -213,28 +189,26 @@ def createTemporalMemoryRegion(network):
 
 
 
-def createClassifierRegion(network, classifierType, classifierParams):
+def _createClassifierRegion(network,
```

Comment:
Why can't these params be `network, regionName, classifierParams`, and then `classifierType` is just within `classifierParams`? Then all the `_createXRegions()` methods take the same args, and if we want we can make a generic `addRegions()` method.